build.gradle

```diff
@@ -60,6 +60,11 @@ allprojects {
                     throw new GradleException("Wildcard dependency forbidden: ${requested.group}:${requested.name}:${requested.version}")
                 }
             }
+            //Fix for showing 100+ errors from javac when using Android Databinding
```

Comment:
I haven't noticed the errors referenced in that issue.  However, it might be worth moving the comment on lines 52 and 53 to above line 57 for clarity.

Reply:
I haven't experienced the issue either, just noticed it while browsing on Dagger's github readme.

Reply:
Ok, got those two comments moved to lines 55 & 56 :+1: